app/controllers/hmis_api/user_controller.rb

```diff
@@ -17,4 +17,9 @@ def index
       email: current_hmis_api_user.email,
     }
   end
+
+  def logout
+    sign_out(:hmis_api_user) # Sign out of the HMIS
+    render json: { success: true }, status: 204
```

Comment:
Since this isn't a json end-point, maybe it should just redirect to `root_path`?

Reply:
Yeah, that'd be fine. This is only used in dev to log out of the HMIS from the warehouse rather than the frontend, so either way works for that purpose. Happy to do whatever you think is best.

Reply:
Resolving as this is addressed by adding a different route that doesn't need this method.